frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -68,20 +55,22 @@ const Header: React.FC = () => {
 
   return (
     <>
-      <AppBar position="static" color="secondary">
+      <AppBar position="static" color="secondary" elevation={0}>
```

Comment:
`elevation` removes the box shadow 
https://stackoverflow.com/questions/34550593/how-to-disable-box-shadows-globally-for-all-material-ui-react-components